packages/experimental/Expander/src/Expander.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { expanderName, ExpanderType, ExpanderProps, ExpanderViewProps, ExpanderChangeEvent } from './Expander.types';
+import { compose, mergeProps, withSlots, UseSlots, buildProps } from '@fluentui-react-native/framework';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const ExpanderComponent = ensureNativeComponent('MSFExpanderView');
+function delay(ms: number) {
+  return new Promise( resolve => setTimeout(resolve, ms) );
+}
+
+export const Expander = compose<ExpanderType>({
+  displayName: expanderName,
+  tokens: [{}, expanderName],
+  slotProps: {
+    root: buildProps(
+      (tokens) => ({...tokens})
+    ),
+  },
+  slots: { root: ExpanderComponent },
+  render: (userProps: ExpanderProps, useSlots: UseSlots<ExpanderType>) => {
+    const Root = useSlots(userProps).root;
+    const [expandedState, setExpandedState] = React.useState(userProps.expanded);
+    const expanderHeight = expandedState? userProps.expandedHeight : userProps.collapsedHeight;
+    async function onExpanderChange(event: ExpanderChangeEvent) {
+      if (event.nativeEvent.expanded != null) {
+        event.persist();
+        if (!event.nativeEvent.expanded) {
+          await delay(175);
+        }
+        setExpandedState(event.nativeEvent.expanded);
```

Comment:
Why do we need the `persist()` call and `delay(175)` call? Maybe add inline comments for future readers as well.

Reply:
When the expander collapses and the height manually changes, the collapsing animation gets cut short, so I added a delay for the height change. I'll add a comment.